templates/partners/gsi.html

```diff
@@ -169,12 +153,11 @@ <h2>
         </li>
       </ul>
     </div>
-    <div class="col-5 u-hide--small u-hide--medium u-align--center u-vertically-center">
-      <img src="https://assets.ubuntu.com/v1/5d8aa78b-1+Discovery.svg" width="335" height="200" alt="">
-    </div>
   </div>
 </section>
 
+{% include '/partners/partial/_co-sell-with-canonical.html' %}
```

Comment:
Ah! I see now, yeah makes sense to make this a partial for sure, but I may be missing some context here. Is the copy supposed to be "Access our partner portal" or "Co-sell with Canonical through the partner portal"? And I assume the image in the mock-up is just a placeholder and not meant to be included yet, is that right?

Reply:
I'm not too sure but last time there was such an image we ended up not including it so I'm just applying the same principles here!